modules/RTC/TraceablePeerConnection.js

```diff
@@ -1495,6 +1543,42 @@ TraceablePeerConnection.prototype.setAudioTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Takes in a *unified plan* offer and inserts the appropriate
+ * parameters for adding simulcast receive support.
+ * @param {} A session description object (with 'type' and 'sdp' fields)
```

Comment:
i did it this way because this is the way we've done all the sdp-munging methods (take in a desc and return the changed one)